octobot_evaluators/data_manager/evaluation.py

```diff
@@ -0,0 +1,43 @@
+#  Drakkar-Software OctoBot-Evaluators
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_commons.evaluators_util import check_valid_eval_note, UNSET_EVAL_TYPE
+from octobot_evaluators.constants import EVALUATOR_EVAL_DEFAULT_TYPE
+
+
+class Evaluation:
```

Comment:
Where it is used ?

Reply:
In strategies to make it easier to store and manipulate evaluations when getting every TA evaluation etc

Reply:
I understand that it brings clarity but I think it's a bit overkill to create an new class at each evaluation **for each** evaluators.

Reply:
Another option would be to use the matrix nodes directly not to create additionnal elements but then we need to be careful never to edit it in any way. Or maybe you have something else in mind ?

Reply:
Yes I agree with an API, but not with creating a lot of object at each evaluation